python/ray/serve/controller.py

```diff
@@ -261,155 +243,74 @@ def get_all_endpoints(self) -> Dict[EndpointTag, Dict[BackendTag, Any]]:
         """Returns a dictionary of backend tag to backend config."""
         return self.endpoint_state.get_endpoints()
 
-    async def _set_traffic(self, endpoint_name: str,
-                           traffic_dict: Dict[str, float]) -> UUID:
-        if endpoint_name not in self.endpoint_state.get_endpoints():
-            raise ValueError("Attempted to assign traffic for an endpoint '{}'"
-                             " that is not registered.".format(endpoint_name))
-
-        assert isinstance(traffic_dict,
-                          dict), "Traffic policy must be a dictionary."
```

Comment:
I'm guessing this is just handled by creating the `TrafficPolicy` object?

Reply:
Yep!